Assets/Scenes/SampleScene.unity

```diff
@@ -1,862 +0,0 @@
-%YAML 1.1
```

Comment:
discard this change if you are only removing spaces it doesn't make sense that you have changed the scene 😅